src/app/states/facade/authFacade.ts

```diff
@@ -0,0 +1,47 @@
+import {Injectable} from '@angular/core';
+import {Store} from '@ngxs/store';
+import {UserLogin, UserLogout, UserRegister, UserErrors, UserChangeLanguage} from '../actions/activeUser.actions';
+import {Observable} from 'rxjs';
+import {User} from '../../interfaces/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthFacade {
+  constructor(
+    private store: Store
+  ) {}
+
+  login(user?: Partial<User>): void {
+    this.store.dispatch(new UserLogin(user));
+  }
+
+  logout(): void {
+    this.store.dispatch(new UserLogout());
+  }
+
+  register(user: Partial<User>): void {
+    this.store.dispatch(new UserRegister(user));
+  }
+
+  addErrors(errors: string) {
+    this.store.dispatch(new UserErrors(errors));
+  }
+
+  getLoggedIn(): Observable<boolean> {
+    return this.store.select(state => state.activeUser.isLoggedIn);
+  }
+
+  getErrors(): Observable<string> {
+    return this.store.select(state => state.activeUser.errors);
+  }
+
+  changeLanguage(lang: string): void {
+    this.store.dispatch(new UserChangeLanguage(lang));
+  }
+
+  getLanguage(): Observable<string> {
+    return this.store.select(state => state.activeUser.language);
```

Comment:
Bestenfalls einen memoized Selector verwenden (https://www.ngxs.io/concepts/select#memoized-selectors):
```suggestion
    return this.store.select(AuthState.language);
```